authmodules/TYPO3/privacyidea/Classes/PrivacyideaService.php

```diff
@@ -110,23 +113,30 @@ public function init() {
 	 * @return int authentication statuscode, one of 0, 100 and 200
 	 */
 	public function authUser(array $user) {
-		// 0 means authentication failure
-		$ret = 0;
-		$username = $this->login['uname'];
-		$password = $this->login['uident_text'];
-		$this->logger->info("try to authenticate user [$username]");
+		if ($this->extConf['privacyIDEABackend'] != 'adminOnly'
```

Comment:
I still do not get, why we need this if? And which condition it should check!

Isn't the authUser Method called, when the init returned True. And if not, this method is not called at all.
So we do not need this if at all. (Was not there in the first place ;-)